README.md

```diff
@@ -418,9 +418,9 @@ The spacing scale is based on `rem` sizes, so it will remain consistent if you c
 List of classes used to create default page grid.
```

Comment:
Github didn't give me the option to suggest on line 416, but should we change from `grid.scss` to `layout.scss`?